src/ci/resources/centaur_application.conf

```diff
@@ -11,7 +11,7 @@ centaur {
   standardTestCasePath: ${CROMWELL_BUILD_CENTAUR_TEST_DIRECTORY}
 
   error-reporter {
-    retry-attempts: 2
+    retry-attempts: 0
```

Comment:
Will this increase the occurrences of flaky tests in Travis?

Reply:
did you mean to commit this?